bin/replication.js

```diff
@@ -115,8 +114,10 @@ class _SetupReplication {
             'default' : sourceProfile;
         const verifyTargetProfile = targetProfile === undefined ?
             'default' : targetProfile;
-        source.auth.vault.host = 'localhost';
-        source.s3.host = 'localhost';
+
+        // TODO: Federation fix made #1125?
+        // source.auth.vault.host = 'localhost';
+        // source.s3.host = 'localhost';
         const destHost = this.destHosts.pickHost().host;
```

Comment:
For this, Rahul wanted to use s3-nginx port 80 https://github.com/scality/backbeat/pull/62#discussion_r133603964